### PR TITLE
Basic Mashup of Google Maps and Meetup.com

### PR DESCRIPTION
**Pull request by msquarednet**

### CODE REVIEW

File: js/collections/meetups.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+//Meetup Collection
+var app = app || {};
+
+(function () {
```
Comment (afeld) [inline js/collections/meetups.js:4]: Not hurting anything, but the self-executing function isn't really doing anything here.

File: js/views/app-view.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
+//App View
+var app = app || {};
+
+(function ($) {
+
+	$.ajaxSetup({
+		headers: {'Access-Control-Allow-Origin':'http://mashup.dev'}
```
Comment (afeld) [inline js/views/app-view.js:7]: Unfortunately, `Access-Control-Allow-Origin` needs to be set server-side in the response to have any effect.

**Comment by afeld** [comment]
Nice work!  Hope that covering OOP in JS from scratch made working with Backbone easier.

Just a piece of feedback, in case you take other classes in the future (though it applies to work done in a company too): it makes things a lot easier to read if you take out the code that's commented out or unused.

Thanks!

File: js/views/app-view.js (hunk 2 above)
Comment (msquarednet) [inline js/views/app-view.js:7]: Good to know. It was worth a shot. :/

**Pull request closed without merging**